Plugins/SwiftLintPlugin/Path+Helpers.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import PackagePlugin
+
+extension Path {
+    /// Scans the receiver, then all of it's parents looking for a configuration file with the name ".swiftlint.yml".
```

Comment:
Nit: `it's` -> `its`

```suggestion
    /// Scans the receiver, then all of its parents looking for a configuration file with the name ".swiftlint.yml".
```